css-ui-4/Overview.bs

```diff
@@ -331,21 +331,21 @@ regardless of color background.
 Conformant UAs may ignore the ''outline-color/invert'' value
 on platforms that do not support color inversion of the pixels on the screen.
 
-If the UA does not support the ''outline-color/invert'' value
+If the UA does not support the ''outline-color/invert'' value,
 then it must reject that value at parse-time, and
 the initial value of the 'outline-color' property
 is the ''color/currentColor'' keyword.
 
-The 'outline' property is a shorthand property,
+The 'outline' property is a shorthand property
 and sets all three of 'outline-style',
 'outline-width',
 and 'outline-color'.
 
 Note: The outline is the same on all sides.
 In contrast to borders,
-there are no ''outline-top'' or ''outline-left'' etc. properties.
+there are no ''outline-top'', ''outline-left'' etc. properties.
```

Comment:
Need another comma between outline-left and etc here.

Reply:
Right. Added.